great_expectations/data_context/types/base.py

```diff
@@ -3,7 +3,7 @@
 import itertools
 import logging
 import uuid
-from copy import deepcopy
+from copy import copy, deepcopy
```

Comment:
How about we just `import copy` (as per the earlier comment)?  Thanks! /cc @roblim